### ISSUE TITLE
【サービス：仮説検証】ユースケース

### ISSUE THREAD

**Issue by urashin** [urashin/micro-volunteer-docs#2]
#1 

本サービスと相性の良い課題を選定し、適した解決方法（サービスの使い方）を考える
- 仮説
  - 障がいから考える
    - 一般的に障がいとして扱われるものに起因した日常的な課題に関する調査
      - デスクトップリサーチ
        - 例：[平成28年生活のしづらさなどに関する調査（全国在宅障害児・者等実態調査）](https://www.mhlw.go.jp/toukei/list/seikatsu_chousa_c_h28.html)
  - 障がい以外で考える
    - 事故・術後の後遺症・持病
      - 例：不安を感じたときに、だれかがすぐにかけつけてくれるような「見守り感」
- 検証
  - 直面している問題と現状の解消コスト
    - 障がいの内容
    - 困る状況
    - 現状の解消方法 or 解消できないことによる損失
  - サービスと相性の良さそうな課題の選定
    - 何が確認できればよいか？
    - バイアスをかけずに確認するためには、どんな質問がよいか？
  - ペルソナ設定
  - 検証過程の情報の扱い
    - 個人情報に近いものも想定されるため、検証の詳細は伏せ、結論だけを記載する

**Comment by urashin** [comment]
@THayashi0000 
現段階である程度見えている[方向性](https://github.com/urashin/micro-volunteer-docs/issues/7#issuecomment-1025720630)に関して、さらに検証を行い、課題の深掘りを行う必要があると思います。
ここで見えている３つの方向性に関して、それぞれインタビュー対象の絞り込み、接点を持つ方法について一緒に考えていただけますか？

**Comment by THayashi0000** [comment]
### 前回ヒアリング結果から本サービスの想定ユーザ
1. 車椅子利用障害者
2. 精神障害者
3. 視覚障害者

### 各ユーザに対する情報整理（前提）と仮説
#### 1. 車椅子利用障害者
（前提）
　- サポートを受けなくてもある程度自分で移動が可能
　- スマホの利用には障壁がない
（仮説）
　- 車椅子を利用していることによる日常的な困りごとに遭遇しやすい
　- 車椅子を利用していることによる日常的な困りごとの中には、周囲の助けなしに解決できないものもある
（疑問点）
　- 誰かに付き添ってもらうときと一人で行動するときの違いは？
　
#### 2. 精神障害者
（前提）
　- サポートを受けなくてもある程度自分で行動が可能
　- スマホの利用には障壁がない
　- 何か想っていても言葉が出てこない時がある
　- 外出中に想定外の問題に遭遇すると、適切に周りに助けを求めることができない
（仮説）
　- 外出中、想定外の問題に遭遇すると、誰かに助けてもらいたいと思うことがある
　- 助けてもらいたいが、相談の方法がわからない
　※別途コミュニティ等を通して調査中
　
#### 3. 視覚障害者 
（前提）
　- 一人で行動・移動することは比較的困難
　- ルーティンワークや、予め予定が立てられる用事については事前にサポートを依頼
　- しかし、夜中に病院にいかなければならないような突発的な問題の場合はサポートを受けられず困ることがある
（仮説）
　- 急用発生時、誰にも助けを求められず、一人で何とかしないと行けない状態がある
（疑問点）
　- スマホをどうやって使いこなしているのか？

**Comment by THayashi0000** [comment]
@urashin 
対象を以下３つに絞った時、まずは車椅子利用障害者に絞ってヒアリングを続けるべきかと思うのですが、どうでしょうか？
1. 車椅子利用障害者
2. 精神障害者
3. 視覚障害者

もし絞る方向でよければ、Slackで担当者に車椅子利用障害者のご紹介が可能か聞いてみようかと思います。
また、同時並行で前回ヒアリングと同じように当日までに、事前調査や検証仮説事項を列挙する予定です。

**Comment by urashin** [comment]
@THayashi0000 はい、スマホアプリ利用に対する障壁が比較的低そう、というところで一旦は車椅子利用者で良いと思います。ソリューションがつかってもらえそうなシーンの設定と、どのような人がそのような場面に多く直面しそうなのかといったペルソナ設定はある程度やっておくと良さそうです。
